templates/web/fixmystreet-uk-councils/report/new/after_category.html

```diff
@@ -1 +0,0 @@
-<p><small>[% loc("Remember that FixMyStreet is primarily for reporting physical problems that can be fixed. If your problem is not appropriate for submission via this site remember that you can contact your council directly using their own website.") %]</small></p>
```

Comment:
I guess we leave it, and we can ask if they want to keep it. Oxfordshire's is specially worded anyway.